packages/handler-kit-azure-func/src/logger.ts

```diff
@@ -0,0 +1,22 @@
+import * as azure from "@azure/functions";
+
+import * as L from "@pagopa/logger";
+
+export const getLogger = (ctx: azure.Context): L.Logger => ({
```

Comment:
can you add a comment with the context? how and when is this method supposed to be used?

Reply:
sure! I'll add the comments immediatly!

this function creates a `L.Logger` implementation (from `@pagopa/logger`) backed by the `azure.Context.logger`